src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -10,72 +9,96 @@ import {
   getActiveExpandedPages,
 } from '../helpers';
 
+import { focusElement } from '../utilities/ui';
+
 import PropTypes from 'prop-types';
 import { REVIEW_APP_DEFAULT_MESSAGE } from '../constants';
 
-export default class FormNav extends React.Component {
-  // The formConfig transforming is a little heavy, so skip it if we can
-  shouldComponentUpdate(newProps) {
-    return !shallowEqual(this.props, newProps);
-  }
```

Comment:
This was a performance optimization, and I removed it because the hooks-equivalent `React.memo` didn't play well with some of our unit tests.